db/migrate/20170604144747_add_foreign_keys_for_accounts.rb

```diff
@@ -0,0 +1,41 @@
+class AddForeignKeysForAccounts < ActiveRecord::Migration[5.1]
+  def change
+    add_foreign_key :statuses, :accounts, on_delete: :cascade
+    add_foreign_key :statuses, :accounts, column: :in_reply_to_account_id, on_delete: :nullify
+    add_foreign_key :statuses, :statuses, column: :in_reply_to_id, on_delete: :nullify
+    add_foreign_key :account_domain_blocks, :accounts, on_delete: :cascade
+    add_foreign_key :conversation_mutes, :accounts, on_delete: :cascade
+    add_foreign_key :conversation_mutes, :conversations, on_delete: :cascade
+    add_foreign_key :favourites, :accounts, on_delete: :cascade
+    add_foreign_key :favourites, :statuses, on_delete: :cascade
+    add_foreign_key :blocks, :accounts, on_delete: :cascade
+    add_foreign_key :blocks, :accounts, column: :target_account_id, on_delete: :cascade
+    add_foreign_key :follow_requests, :accounts, on_delete: :cascade
+    add_foreign_key :follow_requests, :accounts, column: :target_account_id, on_delete: :cascade
+    add_foreign_key :follows, :accounts, on_delete: :cascade
+    add_foreign_key :follows, :accounts, column: :target_account_id, on_delete: :cascade
+    add_foreign_key :mutes, :accounts, on_delete: :cascade
+    add_foreign_key :mutes, :accounts, column: :target_account_id, on_delete: :cascade
+    add_foreign_key :imports, :accounts, on_delete: :cascade
+    add_foreign_key :media_attachments, :accounts, on_delete: :nullify
```

Comment:
>The rule on media attachments table is nullify instead of cascade, because presumably we would want to keep the records so that we have a reference to the files that need to be deleted, otherwise those files would be orphaned.